packages/neuron-wallet/.env

```diff
@@ -0,0 +1,25 @@
+## mainnet
+MAINNET_SUDT_DEP_TXHASH=0x0000000000000000000000000000000000000000000000000000000000000000
+MAINNET_SUDT_DEP_INDEX=0
+MAINNET_SUDT_DEP_TYPE=code
+MAINNET_SUDT_SCRIPT_CODEHASH=0x0000000000000000000000000000000000000000000000000000000000000000
+MAINNET_SUDT_SCRIPT_HASHTYPE=data
+MAINNET_SUDT_HEIGHT=-1
+MAINNET_ACP_DEP_TXHASH=0x0000000000000000000000000000000000000000000000000000000000000000
+MAINNET_ACP_DEP_INDEX=0
+MAINNET_ACP_DEP_TYPE=code
+MAINNET_ACP_SCRIPT_CODEHASH=0x0000000000000000000000000000000000000000000000000000000000000000
```

Comment:
Do we have to set these variables to a dummy values if they are not available yet on mainnet?
Can we just comment them out for now?

Reply:
I think it's better to leave them `NULL` value if these values are not available yet. Setting them to a dummy value like `0x0000000000000000000000000000000000000000000000000000000000000000 ` or `-1` could be misleading.

Reply:
`null` will cause type mismatch error

Reply:
Then we can change the type to be nullable.

Reply:
After a detailed look at the codes coupled with these variables, it does need some works to refactor these values to `nulls`. 

I will create an issue to remind the refactoring of these variables in a later stage.